app/Listeners/Mail/StoreMailInDatabase.php

```diff
@@ -38,7 +38,7 @@ public function handle(MessageSent $event)
             'cc' => $this->structureAddressData($event->message->getCc()),
             'bcc' => $this->structureAddressData($event->message->getBcc()),
             'subject' => $event->message->getSubject(),
-            'body' => $event->message->getTextBody(),
+            'body' => $event->message->getHtmlBody() ?? $event->message->getTextBody(),
```

Comment:
We expect body to be returned by `getHtmlBody` but mail can be created another way (not sure it is in GPM, though) that sets text body, so if we don't have html body contents get text body.